application/src/main/resources/application.yml

```diff
@@ -18,3 +14,8 @@ dynamo:
 
 invalid:
   postcode: Please enter a valid postcode
+
+api:
+  version: 1.0
+  title: NHS Postcode Mapping Application Interface
+  description: NHS Postcode Mapping APIs provide NHS Services information to the users
```

Comment:
Maybe change to:
NHS Postcode Mapping API provides northing and easting location coordinates for a given postcode.

Reply:
done